docs/verify/simulate-with-qiskit-primitives.ipynb

```diff
@@ -0,0 +1,584 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "235b9def-4c99-4511-87dc-fe972acd1249",
+   "metadata": {},
+   "source": [
+    "# Exact simulation with Qiskit primitives"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "eda9656b-3403-40e3-a466-6007466adb93",
+   "metadata": {},
+   "source": [
+    "The reference primitives in Qiskit perform local statevector simulations. These simulations do not support\n",
+    "modeling device noise, but are useful for quickly prototyping algorithms before looking into more advanced simulation\n",
+    "techniques (using Qiskit Aer) or running on real devices (Qiskit Runtime primitives).\n",
+    "\n",
+    "The `Estimator` primitive can compute expectation values of circuits, and the `Sampler` primitive can sample\n",
+    "from output distributions of circuits.\n",
+    "\n",
+    "The following sections show how to use the reference primitives to run your workflow locally."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "343d1fc6-f999-469c-9708-7536c3238b08",
+   "metadata": {},
+   "source": [
+    "## Use the reference `Estimator`\n",
+    "\n",
+    "There are two reference implementations of `Estimator` in `qiskit.primitives` that run on a local statevector\n",
+    "simulators: the [`StatevectorEstimator`](../api/qiskit/qiskit.primitives.StatevectorEstimator) class and the\n",
+    "[`Estimator`](../api/qiskit/qiskit.primitives.Estimator) class. The [`StatevectorEstimator`](../api/qiskit/qiskit.primitives.StatevectorEstimator)\n",
+    "implements the new Estimator V2 interface introduced in the Qiskit SDK 1.0, and offers additional input vectorization\n",
+    "features in comparison with the [`Estimator`](../api/qiskit/qiskit.primitives.Estimator) class, which implements the\n",
+    "legacy Estimator V1 interface. Both can take circuits, observables, and parameters as inputs and return the locally\n",
+    "computed expectation values.\n",
+    "\n",
+    "The following code prepares the inputs that will be used in the examples that follow. The expected input type for the\n",
+    "observables is [`qiskit.quantum_info.SparsePauliOp`](../api/qiskit/qiskit.quantum_info.SparsePauliOp). Note that\n",
+    "the circuit in the example is parametrized, but you can also run the Estimator on non-parametrized circuits.\n",
+    "\n",
+    "<Admonition type=\"note\">\n",
+    "  Any circuit passed to an Estimator must **not** include any **measurements**.\n",
+    "</Admonition>"
```

Comment:
2 notes on admonitions on Jupyter notebooks:

1. the admonition syntax used in mdx files doesn't work
2. the standard emphasis syntax (`**bold**`) doesn't work inside the admonitions, you need to specify it in html (`<b>bold<\b>`)

I normally use the following syntax to create these "admonition style cells". There are 4 possible colors for different alert types, you can find them in this cheatsheet: https://www.ibm.com/docs/en/watson-studio-local/1.2.3?topic=notebooks-markdown-jupyter-cheatsheet (look for "Colored note boxes").

```suggestion
    "<div class="alert alert-block alert-info">\n",
    "  Any circuit passed to an Estimator must <b>not</b> include any <b>measurements</b>.\n",
    "</div>"
```